assets/js/trento.jsx

```diff
@@ -45,8 +45,11 @@ function App() {
 
   return (
     <Provider store={store}>
-      <Toaster position="top-right" />
       <BrowserRouter>
+        <Toaster
```

Comment:
As commented, we need to have this to be able to use `Link` in the toasts (everything router related)